ajax/libs/jquery.flipster/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "jquery.flipster",
+  "version": "1.1.3",
+  "homepage": "http://brokensquare.com/Code/jquery-flipster/demo/",
+  "description": "Responsive coverfpsterlow-style carousel plugin for jQuery",
+  "filename": "jquery.flipster.min.js",
+  "author": "Adrien Delessert <a@drien.com>",
+  "keywords": [
+    "coverflow",
+    "jquery",
+    "carousel",
+    "flipster",
+    "jquery-plugin",
+    "ecosystem:jquery"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/drien/jquery-flipster.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/drien/jquery-flipster.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "+(js|css)/*.+(js|css|map)",
```

Comment:
This is for v0.2.0 ~ v0.3.5. I think `+(js|css)/jquery.flipster*` is a better option.